README.md

```diff
@@ -1,53 +1,31 @@
 # Beluga
 
-![CI badge](https://github.com/Ekumen-OS/beluga/actions/workflows/ci_pipeline.yml/badge.svg?event=push)
+[![CI pipeline](https://github.com/Ekumen-OS/beluga/actions/workflows/ci_pipeline.yml/badge.svg?branch=main)](https://github.com/Ekumen-OS/beluga/actions/workflows/ci_pipeline.yml?query=branch:main)
+[![License Apache-2.0](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](LICENSE)
 
 ## Overview
 
-Beluga is an extensible library with a ground-up implementation of the Monte Carlo Localization (MCL) family of estimation algorithms,
-including sensor and motion models.
-It also provides a _drop-in_ replacement for the `amcl` node used in [navigation2](https://github.com/ros-planning/navigation2).
+Beluga is an extensible C++ library with a ground-up implementation of the Monte Carlo Localization (MCL) family of estimation algorithms featuring:
```

Comment:
```suggestion
Beluga is an extensible C++17 library with a ground-up implementation of the Monte Carlo Localization (MCL) family of estimation algorithms featuring:
```

Reply:
Done in https://github.com/Ekumen-OS/beluga/pull/186/commits/570081679de0004a1a82d9ef2f29b2e232069b48.